api/nakadi-event-bus-api.yaml

```diff
@@ -800,18 +800,22 @@ paths:
       security:
         - oauth2: ['nakadi.event_stream.read']
       description: |
-        Endpoint for committing offsets of the subscription. The offsets are per partition of EventType. One
-        can use this endpoint to commit the offsets for any of the partitions that are part of this
-        subscription.
-        With every commit the offsets can only increase so it is not possible to commit offset that is lower
-        than already committed offset for the appropriate partition; if such request is made - it will not
-        have any effect on the committed offsets.
+        Endpoint for committing offsets of the subscription.
+        The behavior of commit is specific to `commit_mode` and `commit_semantics` parameters of the session
+        that makes a commit. Please see details in subscription streaming endpoint specification.
```

Comment:
I'm not really sure about that. I think that clarifies that commit is done by session, If we remove it than it will not be clear enough.